app/views/curation_concerns/base/_form_access_policy.erb

```diff
@@ -14,6 +14,10 @@
   <%= f.input :rights_statement,
               as: :select,
               collection: RightsService.select_options,
-              input_html: { class: 'form-control' },
+              input_html: { class: 'form-control rights-statement' },
               label: 'Rights Statement' %>
+
+  <% readonly = '1' unless RightsStatementService.notable?(@curation_concern.rights_statement) %>
+  <%= f.input :rights_note, as: :text, label: 'Rights Note',
+      input_html: { class: 'rights-note', readonly: readonly }  %>
```

Comment:
Can we move the readonly logic to the presenter by doing something like readonly: `f.object.notable_rights_statement?`? Might make this a little easier to switch up later if we need to.